src/components/FormSubmit/index.tsx

```diff
@@ -1,23 +1,23 @@
-import lodashGet from 'lodash/get';
-import React, {useEffect} from 'react';
+import React, {KeyboardEvent as ReactKeyboardEvent, useEffect} from 'react';
```

Comment:
NAB: Is it necessary to rename KeyboardEvent? Maybe let's keep it simple if there are no conflicts